units/Princess.cfg

```diff
@@ -60,4 +62,118 @@
         [/frame]
         {SOUND:HIT_AND_MISS {SOUND_LIST:SWORD_SWISH} {SOUND_LIST:MISS} -75}
     [/attack_anim]
+
+    [variation]
+        variation_id=spirit
+        inherit=yes
+        variation_name= _ "her soul"
```

Comment:
I am not sure about the variation name. I would use Soul or Spirit (that is without personal pronoun and with first capital letter). But this is not so important.